docs/rules/prefer-name-spaced-react.md

```diff
@@ -0,0 +1,36 @@
+# Prefer React import be name-spaced. (prefer-name-spaced-react)
```

Comment:
did you meant of ill this in?

Reply:
oh yes, sorry, I forgot i was just looking for general approach approval at first. Will clean this all up.